src/config/config.ts

```diff
@@ -0,0 +1,77 @@
+/*
+    Copyright 2019-2020 eMobilify GmbH
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+*/
+import * as uuid from "uuid"
+import { IOcnToolsConfig } from "../types"
+
+export const config: IOcnToolsConfig = {
+    ocn: {
+        node: "https://test-ocn.emobilify.com",
+        stage: "volta"
+    },
+    cpo: {
+        port: 3000,
+        publicIP: "https://dev.ev-dashboard-cpo.energyweb.org",
```

Comment:
Please use `process.env` to setup configs

Reply:
Will be changed after it starts working